### PR TITLE
TQ42-1247

### PR DESCRIPTION
**Pull request by lusnig**
Update documentation for Time Series Prediction with categorical or Boolean output

Added documentation about Metrics and Loss functions for:

1. MSE
2. MAE
3. Boolean Cross Entropy (BCE)
4. Categorical Crossentropy (CROSSENTROPY)

### CODE REVIEW

File: docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md [hunk 1]
```diff
@@ -114,3 +114,123 @@ The Hybrid Quantum Long Short-Term Memory (HQLSTM) is an advanced version of the
...
+  
+### Loss functions  
+A loss function is a function that compares the output values predicted by the network with the target values provided by the user.   
+It estimates how well the neural network models the training data and during the training phase, the aim is to minimise the loss between the expected and predicted output values.  
+In supervised learning, the loss functions are classified into two classes according to the type of learning task: regression and classification loss functions  
+  
+##### Regression
+A regression loss function predicts a continuous quantity  
+  
+###### Mean Square Error (MSE)
```
Comment (sbtq) [inline docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md:127]: Missing a `d` it should be `Mean Squared Error`, right?

File: docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md [hunk 2]
```diff
@@ -114,3 +114,123 @@ The Hybrid Quantum Long Short-Term Memory (HQLSTM) is an advanced version of the
...
+  
+##### F1-score  
+F1 Score is the Harmonic Mean between precision and recall and indicates how precise and robust is the classifier  
+  
+$F1 = 2*\frac{1}{\frac{1}{precision}+\frac{1}{recall}}$  
+  
+#### Regression   
+Regression models have a continuous output so the metrics are based on calculating some sort of distance between the prediction and the ground truth.  
+  
+#### Mean Square Error
```
Comment (sbtq) [inline docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md:231]: See above

File: docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md (hunk 1 above)
Comment (lusnig) [reply docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md]: sorry for the typo.
added it

File: docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md (hunk 2 above)
Comment (lusnig) [reply docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md]: sorry for the typo.
added it

**Review by rbattq — APPROVED** [review]

**Review by sbtq — APPROVED** [review]
